Rakefile

```diff
@@ -87,9 +87,10 @@ task 'dlls', [:plat] do |t, args|
   plat_list = args[:plat]
 
   build_configs = []
+  w64_ucrt = { cross: 'x86_64-w64-mingw32', out: 'grpc_c.64-ucrt.ruby', platform: 'x64-mingw-ucrt' }
```

Comment:
AFAIK the rake-compiler-dock [docker image for ucrt](https://github.com/rake-compiler/rake-compiler-dock/blob/master/mingw64-ucrt/Dockerfile) uses the same gcc/g++ packages as the x64-mingw image, and patches them(?)